src/courses/advanced/03.md

```diff
@@ -168,7 +168,9 @@ end
 
 ### Running the Controls
 
-To run `inspec exec` on the target, ensure that you are in the directory that has `my_nginx` profile.
+:::tip
+To run `inspec exec` on the target, ensure that you are in the directory that has the `my_nginx` profile. If you're using the inputs file that you generated in the User course, we recommend renaming the file to `inputs-linux.yml` to match examples in the Advanced course. Additionally, if the inputs file is not in the root of the `my_nginx` directory, you will need to specify its filepath in the `--input-file` flag or move the file to the root of the profile directory.
```

Comment:
@wdower Adding this when necessary in advanced course instead of arbitrarily in the user course. Also addresses the filepath issue that the student ran into today.